modules/processing/network.py

```diff
@@ -111,6 +111,21 @@ def __init__(self, filepath):
         self.irc_requests = []
         # Dictionary containing all the results of this processing.
         self.results = {}
+        # List containing all whitelist entries.
+        self.whitelist = self._build_whitelist()
+
+    def _build_whitelist(self):
+        result = []
+        whitelist_path = os.path.join(
+            CUCKOO_ROOT, "data", "whitelist", "domain.txt"
+        )
+        f = open(whitelist_path, 'rb')
```

Comment:
@doomedraven sure - I'll verify that also just in case later today and commit when load drops off my box